### ISSUE TITLE
confusing data being written to S3 from the firehose

### ISSUE THREAD

**Issue by ciaranlee** [awslabs/lambda-streams-to-firehose#46]
Hi!

I've got this function up and running - processing a dynamodb stream in to a kinesis firehose which then writes to S3. The only data I'm seeing in S3 is endless lines of this:

```
[object Object]
[object Object]
[object Object]
[object Object]
[object Object]
[object Object]
[object Object]
```

Any idea what I need to do in order to get meaningful data relating to the changes to DynamoDB records written out to S3?

Thanks!

**Comment by IanMeyers** [comment]
This is because a pull request was merged with broke a transformer. Working on the fix will will be in 1.4.5 now...

**Comment by IanMeyers** [comment]
This is now fixed in 1.4.5

**Comment by ciaranlee** [comment]
Awesome! Thanks @IanMeyers :)
